internal/storage/driver/postsql/binding_test.go

```diff
@@ -28,7 +28,7 @@ func TestBinding(t *testing.T) {
 		assert.NoError(t, err)
 
 		// when
-		createdBinding, err := brokerStorage.Bindings().GetByBindingID(testBindingId)
+		createdBinding, err := brokerStorage.Bindings().Get("instance-"+testBindingId, testBindingId)
```

Comment:
If we allow to have instances with multiple bindings, this manner of creating instanceID seems not appropriate.

Reply:
For not it is not necessary. Further methods will be added in PRs associated with their functionality.